### DISCUSSION TITLE
Move tags to Content Settings

### DISCUSSION THREAD

**Discussion by parthlawate** [joomla/joomla-cms#32618] [2. Cold Features]
While Both Fields, Categories and tags are in effect 'Infrastructure' extensions available in Joomla that can be used by any extension, We show categories & Field related menus under the 'Joomla Content' in admin. 

Generally Fields, Tags and Categories are all building blocks of Taxonomy. So it would make sense that we Also Link "Tags" under Content Settings.  But for a user new to Joomla this will make things easier for them..

It can stay on as a separate menu for the general context. 

![Screenshot 2021-03-09 at 12 16 51 PM](https://user-images.githubusercontent.com/673176/110429720-5820d900-80d1-11eb-90e4-01fe03db4e4f.png)

**Comment by brianteeman** [comment]
> Generally Fields, Tags and Categories are all building blocks of Taxonomy.

What do fields have to do with taxonomy?

I've been staring at the screenshot for ages and I can't see what you are trying to show

**Reply by parthlawate** [reply]
Hi Brian, 

Fields can be used to extend your core content to add more structured information .. Some of that can be classification information when you use them as 'Filters'. They can in effect do the same job as categories and tags do.. allow you to classify content in multiple ways and introduce new facets to navigating content. 

The screenshot is just given to show how fields are under content settings.. And that tags could go some where there so that the end user knows he has these 3 systems to work with (tags, categories & fields)

**Reply by brianteeman** [reply]
That is just one use of fields (not one I have ever used myself) and not something that can actually be done with core

**Comment by infograf768** [comment]
I guess what the OP wants is adding Tags in that dashboard view

<img width="1244" alt="Screen Shot 2021-03-09 at 10 06 04" src="https://user-images.githubusercontent.com/869724/110446176-2dc62000-80bf-11eb-80b9-0f223d434d67.png">


which is possible by adding
```
	<menuitem
		title="COM_TAGS"
		type="heading"
		icon="tags"
		class="class:file-alt"
		dashboard="content"
		>
		<menuitem
			title="COM_TAGS"
			type="component"
			element="com_tags"
			link="index.php?option=com_tags&amp;view=tags"
			class="class:image"
		/>
	</menuitem>
```
in `/administrator/components/com_content/presets/content.xml`

**Reply by brianteeman** [reply]
Tags do not have context and can be used in any supported component

**Reply by parthlawate** [reply]
Yes Brian tags don't have context..  

However its a good tool to have for a Content manager in the context of Joomla Content. So I was suggesting also adding it there while maintaining its independent location ..

**Reply by simbus82** [reply]
Backend users, editors, copywriters and marketers don't care about developer logical constraints.
As in any content management system, they expect all the elements to manage content, including the "tags".

Thanks @parthlawate for your catch!
